analog-commander/usr/share/asteroid-launcher/watchfaces/analog-commander.qml

```diff
@@ -0,0 +1,499 @@
+                                              "#00D0E562" :
+                                              "#0023F0C7"
+                                      )
+                gradient.addColorStop(0.97,
+                                      batteryChargePercentage.percent < 30 ?
+                                          "#ffEF476F" :
+                                          batteryChargePercentage.percent < 60 ?
+                                              "#ffD0E562" :
+                                              "#ff23F0C7"
+                                      )
+                ctx.lineWidth = root.height * 0.005
+                ctx.lineCap="round"
+                ctx.strokeStyle = gradient
+                ctx.beginPath()
+                ctx.arc(parent.width / 2,
+                        parent.height / 2,
+                        parent.width * 0.456,
+                        270 * rad,
+                        ((batteryChargePercentage.percent/100*360)+270) * rad,
+                        false
+                        );
+                ctx.lineTo(parent.width / 2,
+                           parent.height / 2)
+                ctx.stroke()
+                ctx.closePath()
+            }
+        }
+
+        Text {
+            z: 2
+            id: batteryDisplay
+			anchors {
+				centerIn: parent
+            	verticalCenterOffset: -parent.height * -0.0200 
+			}	
+            renderType: Text.NativeRendering
+			font {
+				pixelSize: parent.height * 0.39
+            	family: "Teko"
+            	styleName:"Light"
+			}
+            color: "#ffffffff"
+            text: batteryChargePercentage.percent
+            opacity: !displayAmbient ? 0.8 : 0.3
+
+            Text {
+                 z: 9
+                 id: batteryPercent
+                 anchors {
+                     centerIn: batteryDisplay
+                     verticalCenterOffset: parent.height*0.34
+                 }
+                 renderType: Text.NativeRendering
+                 font.pixelSize: parent.height * 0.194
+                 font.family: "Teko"
+                 font.styleName:"Regular"
+                 lineHeightMode: Text.FixedHeight
+                 lineHeight: parent.height * 0.94
+                 horizontalAlignment: Text.AlignHCenter
+                 color: !displayAmbient ?
+                            "#bbffffff" :
+                            "#55ffffff"
+                 text: "BAT<br>%"
+             }
+        }
+    }
+
+    layer.enabled: true
+    layer.effect: DropShadow {
+        transparentBorder: true
+        horizontalOffset: 2
+        verticalOffset: 2
+        radius: 5.0
+        samples: 8
+        color: "#99000000"
+    }
+
+    Item {
+        id: handBox
+
+        z: 3
+        anchors.fill: parent
+
+        Image {
+            id: hourSVG
+
+            z: 3
+            source:imgPath + "hour.svg"
+            anchors.fill: parent
+
+            transform: Rotation {
```

Comment:
I think that perhaps when you resized the hands, they are no longer centered.  As you can see from this picture when the watch was set to 6:00:00 both the hour hand and the minute hand are offset from center.  You could either alter the SVG picture for each hand (which I'd recommend) or you could add a `Translation` to this transform immediately before the `Rotation`.
![hands offset](https://user-images.githubusercontent.com/4911249/177016239-5b3d1bc7-ba6a-459d-8226-76b673a77356.png)